src/index.html

```diff
@@ -258,3 +170,27 @@ <h3 class="text-white d-block my-4">Stay up to date</h3>
     </p>
   </div>
 </section>
+
+<svg
+  style="visibility: hidden; position: absolute;"
+  width="0"
+  height="0"

```

Comment:
What is this SVG for?

Reply:
It makes the rhombus have nice rounded corners.
<img width="1512" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/1da95663-8f3c-423f-a749-f25e6433adc8">

If you take the `rhombus-parent` class out, the rhombus has sharp edge corners:
<img width="1512" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/1985c7d3-5d5d-4c78-96c7-1980c0feec7f">

Reply:
In subsequent PRs, I will put the SVG into an `includes` so we can use it across the app for all pages that have a rhombus.

Reply:
Seems to be accomplished with the [CSS `filter` property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) - very cool!